recipe/meta.yaml

```diff
@@ -8,23 +8,21 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/platformdirs-{{ version }}.tar.gz
-  sha256: 367a5e80b3d04d2428ffa76d33f124cf11e8fff2acdaa9b43d545f5c7d661ef2
+  sha256: 58c8abb07dcb441e6ee4b11d8df0ac856038f944ab98b7be6b27b2a3c7feef19
 
 build:
   number: 0
-  noarch: python
+  skip: True  # [py<37]
```

Comment:
shouldn't this be py<36 ?

Reply:
Nope, see https://github.com/platformdirs/platformdirs/blob/60fc40c294d90f7687f779487b8c19679fb73255/pyproject.toml#L20

Reply:
python 3.6 support was dropped in v2.4.1 https://github.com/platformdirs/platformdirs/blob/2.5.2/CHANGES.rst#platformdirs-241-2021-12-26